src/main/static/stylesheets/components.scss

```diff
@@ -267,3 +267,21 @@
     justify-content: center;
   }
 }
+
+.accordion {
+  padding: 40px 20px 40px 50px;
+
+  &__header {
+    display: flex;
+
+    &__button {
+      margin-left: auto;
```

Comment:
I think this is better expressed as 

&__header {
  display: flex;
  justify-content: space-between;
}